frontend/src/components/FamilyTree/FamilyTreeEdge.tsx

```
@@ -0,0 +1,128 @@
+  const pathData = useMemo(() => {
+    const midX = (sourceX + targetX) / 2;
+    const midY = (sourceY + targetY) / 2;
+
+    switch (relationType) {
+      case 'parent-child':
+        // Straight line for parent-child relationships
+        return `M${sourceX},${sourceY} L${targetX},${targetY}`;
+      
+      case 'spouse':
+        // Curved line for spouse relationships
+        const controlX = midX;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)